pkg/util/constant.go

```diff
@@ -72,6 +72,9 @@ const (
 	HostedClusterScoresResourceName = "hosted-clusters-score"
 	// AddOnPlacementScore score name
 	HostedClusterScoresScoreName = "hostedClustersCount"
+
+	// Maximum hosted cluster count on a hosting cluster
+	MaxHostedClusterCount = 80
```

Comment:
I feel like we need to make this the default, but allow an environment configuration that we can apply or use the ConfigMap.  It's 80 for now, but say they want to move to 100 and say that we want 80 in AWS and 160 in Azure down the road.

Reply:
After Phil completes the taint/node selector work, I can add the addondeploymentconfig to make it configurable